packages/framework/src/Resources/views/Common/Inline/Icon/icon.html.twig

```diff
@@ -0,0 +1,8 @@
+{% if type == 'svg-font' %}
+    <i class="svg svg-{{ name }} {{ attr.class|default('')}}"
```

Comment:
Could we get rid of the space between `svg-{{ name }}` and `{{ attr.class|default('')}}"`. I am not sure if it has any impact (e.g. w3c standards), but is weird for my eyes.
When we rework classes to be list we could solve it by using `|join()` 😉.

Reply:
Well you need space between those strings, otherwise they would be joined.

Reply:
I udestand that. Anyway you do not need it when there is no `attr.class` specified. Then it will result in:

```
 <i class="svg svg-doge "></i>
```